gravann/_io.py

```diff
@@ -103,7 +136,55 @@ def save_plots(model, encoding, mascon_points, lr_log, loss_log, weighted_averag
     plt.ylabel("Loss")
     plt.legend(["Loss", "Weighted Average Loss",
                 "Vision Loss"])
-    plt.savefig(folder+"loss_plot.png", dpi=150)
+    plt.savefig(folder + "loss_plot.png", dpi=150)
+    print("Done.")
+
+    print("Creating LR plot...")
+    plt.figure()
+    abscissa = np.cumsum(n_inferences)
+    plt.semilogy(abscissa, lr_log)
+    plt.xlabel("Thousands of model evaluations")
+    plt.ylabel("LR")
+    plt.savefig(folder + "lr_plot.png", dpi=150)
+
+
+def save_plots_v2(model, encoding, sample, lr_log, loss_log, weighted_average_log, n_inferences,
```

Comment:
can we give the two save_plots a bit more verbose names on what is different? :D